tests/samples/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ iree_lit_test_suite(
     "pack_pipeline_funcIR.mlir"
     "pack_pipeline_funcIR_2core.mlir"
     "pack_pipeline_funcIR_4core.mlir"
```

Comment:
I feel there are too many tests here for this task. Can you move some of them into one file?

Reply:
Good point, I'll take care of that!

Reply:
Just a head-up you might run into a problem with splitting the single file into multiple tests described over here https://github.com/nod-ai/iree-amd-aie/pull/134

Reply:
Thanks for the warning, yes I have noticed this issue with splitting as well but so far did not have the time to find a solution. Will give it another try ;)

Reply:
@nirvedhmeshram I don't get why it's not working with --split-input-file flag? It seems to work fine when I added it in this test https://github.com/nod-ai/iree-amd-aie/blob/main/tests/samples/simple_pack_pipeline_e2e.mlir#L1

Reply:
@yzhang93 the difference is that in the test you linked, you run the same command on two different inputs which I have also used in previous tests, but the tests for 2 cores and 4 cores check different commands (the flag num-cores changes)

Reply:
I integrated these three tests into one now!

Reply:
Thanks for making the tests into one file! Since you only change the M input size, I don't think you need to add this test "pack_pipeline_funcIR_4core_256_256_256.mlir". Just trying to keep minimal number of tests in this folder.

Reply:
I agree, removed it now.